src/common/options/mgr.yaml.in

```diff
@@ -15,7 +15,7 @@ options:
 - name: mgr_pool
   type: bool
   level: dev
-  desc: Allow use/creation of .mgr pool.
+  desc: Allow use/creation of the ``.mgr`` pool. Hey, hacker, leave this value alone.
```

Comment:
I advocate with what probably amounts to my dying breath for allowing this Pink Floyd reference to stand.